contracts/staking/contracts/src/fees/MixinExchangeFees.sol

```diff
@@ -198,7 +198,7 @@ contract MixinExchangeFees is
 
         // sanity check - this is a gas optimization that can be used because we assume a non-zero
         // split between stake and fees generated in the cobb-douglas computation (see below).
-        if (totalFeesCollected == 0 || totalWeightedStake == 0) {
```

Comment:
What was the purpose of this? Seems to just add more logic.

Reply:
We may no longer need this with the new math. But these were denominators in the old cobb-douglas implementation, so this avoided any division by zero.

Reply:
I mean, why remove the check?

Reply:
Ah ~ if there's no stake in a given pool then the entire reward goes to the operator. Similarly, if no pools are staked then the reward goes entirely to operators. Note that it's still in a pool's best interest to stake since alpha != 0. So if all the pools are unstaked, then one pool could come along with relatively low volume but non-zero stake and take a good portion of the total reward.